### PR TITLE
Queue | Deactivate Case Details link for paper cases

### LINKED ISSUE #5394: Deactivate Case Details link for paper cases
When an attorney tried to open a paper case (BVA decision date of 1999) from the Case List, the page did not load. I checked in VACOLS and the case was indeed a paperless case. I suggest we put together an error state, or deactivate the link for paper cases. 

## AC
- When the case is paperless, deactivate the Case Details link.
- Underneath the Veteran's name, like where we have "The Veteran is not the appellant", include the copy:
   - This is a paper case.

### PR DESCRIPTION
**Pull request by dannysellers**
Resolves #5394 

### Description
This disables the case details link and displays a message for paper cases

### Acceptance Criteria 
- [ ] Paper cases have no link to appeal detail view, say "This is a paper case" underneath disabled link

### Testing Plan
1. Re-seed FACOLS (`rake local:vacols:seed`)
2. Go to `/queue` as `BVAAABSHIRE`
3. Confirm `Nola L Stroman (994806951)` link is disabled, paper case note appears
4. Search for `994806951` in case search, confirm Nola Stroman is returned, link is disabled, paper case note appears

### CODE REVIEW

File: app/models/serializers/work_queue/appeal_serializer.rb [hunk 1]
```diff
@@ -50,6 +50,9 @@ class WorkQueue::AppealSerializer < ActiveModel::Serializer
   attribute :status
   attribute :decision_date
   attribute :certification_date
+  attribute :paper_case do
+    object.file_type.eql? "Paper"
```
Comment (dannysellers) [inline app/models/serializers/work_queue/appeal_serializer.rb]: Appeal `file_type` set by [`AppealRepository.folder_type_from`](https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/services/appeal_repository.rb#L216)

File: client/app/queue/constants.js [hunk 2]
```diff
@@ -42,6 +43,10 @@ export const redText = css({ color: '#E60000' });
 export const boldText = css({ fontWeight: 'bold' });
 export const fullWidth = css({ width: '100%' });
 export const dropdownStyling = css({ minHeight: 0 });
+export const subHeadTextStyle = css({
+  fontSize: 'small',
+  color: COMMON_COLORS.GREY_MEDIUM
+});
```
Comment (lowellrex) [inline client/app/queue/constants.js]: 🙌

File: client/app/queue/CaseDetailsLink.jsx [hunk 3]
```diff
@@ -1,38 +1,41 @@
...
+        disabled={!task.task_id || appeal.paper_case}
         onClick={this.setActiveAppealAndTask}
       >
-        {this.props.appeal.attributes.veteran_full_name} ({this.props.appeal.attributes.vbms_id})
+        {appeal.veteran_full_name} ({appeal.vbms_id})
       </Link>
-      {!_.isNull(_.get(this.props.appeal.attributes, 'appellant_full_name')) && <React.Fragment>
+      {!_.isNull(_.get(appeal, 'appellant_full_name')) && <React.Fragment>
+        <br />
+        <span {...subHeadTextStyle}>Veteran is not the appellant</span>
```
Comment (lowellrex) [inline client/app/queue/CaseDetailsLink.jsx]: Do you have any interest in moving these strings into `COPY.json`? Could looks something like the following (don't have any strong preference about what we name them):
```
import COPY from '../../../COPY.json';
...
<span {...subHeadTextStyle}>{COPY.DIFFERENT_VETERAN_AND_APPELLANT_MESSAGE}</span>
...
<span {...subHeadTextStyle}>{COPY.IS_PAPER_CASE_MESSAGE}</span>
```

We could then use those variables [in the tests](https://github.com/department-of-veterans-affairs/caseflow/pull/5550/files#diff-9ae8a8f691260bcfd0720571139be89aR387) with `COPY::DIFFERENT_VETERAN_AND_APPELLANT_MESSAGE` and `COPY::IS_PAPER_CASE_MESSAGE`.

Comment (lowellrex) [reply client/app/queue/CaseDetailsLink.jsx]: We could also reuse that variable in the [CaseListTable as well](https://github.com/department-of-veterans-affairs/caseflow/pull/5550/files#diff-6080bf133dbfe84da0f7ba991d5c2736R50).

**Pull request merged**